docs/_posts/murat-gunay/2021-12-29-classifierdl_xlm_roberta_sentiment_sw.md

```diff
@@ -0,0 +1,155 @@
+---
+layout: model
+title: Sentiment Analysis of Swahili texts
+author: John Snow Labs
+name: classifierdl_xlm_roberta_sentiment
+date: 2021-12-29
+tags: [swahili, sentiment, sw, open_source]
+task: Sentiment Analysis
+language: sw
+edition: Spark NLP 3.3.4
+spark_version: 3.0
+supported: true
+article_header:
+  type: cover
+use_language_switcher: "Python-Scala-Java"
+---
+
+## Description
+
+This model identifies the sentiments (positive or negative) in Swahili texts.
+
+## Predicted Entities
+
+`Negative`, `Positive`
+
+{:.btn-box}
+<button class="button button-orange" disabled>Live Demo</button>
+<button class="button button-orange" disabled>Open in Colab</button>
+[Download](https://s3.amazonaws.com/auxdata.johnsnowlabs.com/public/models/classifierdl_xlm_roberta_sentiment_sw_3.3.4_3.0_1640766370034.zip){:.button.button-orange.button-orange-trans.arr.button-icon}
+
+## How to use
+
+
+
+<div class="tabs-box" markdown="1">
+{% include programmingLanguageSelectScalaPythonNLU.html %}
+```python
+document_assembler = DocumentAssembler() \
+      .setInputCol("text") \
+      .setOutputCol("document")
+
+tokenizer = Tokenizer() \
+      .setInputCols(["document"]) \
+      .setOutputCol("token")
+    
+normalizer = Normalizer() \
+      .setInputCols(["token"]) \
+      .setOutputCol("normalized")
+
+stopwords_cleaner = StopWordsCleaner.pretrained("stopwords_sw", "sw") \
+        .setInputCols(["normalized"]) \
+        .setOutputCol("cleanTokens")\
+        .setCaseSensitive(False)
+
+embeddings = XlmRoBertaEmbeddings.pretrained("xlm_roberta_base_finetuned_swahili", "sw")\
+    .setInputCols(["document", "cleanTokens"])\
+    .setOutputCol("embeddings")
+
+embeddingsSentence = SentenceEmbeddings() \
+      .setInputCols(["document", "embeddings"]) \
+      .setOutputCol("sentence_embeddings") \
+      .setPoolingStrategy("AVERAGE")
+
+sentimentClassifier = ClassifierDLModel.pretrained("classifierdl_xlm_roberta_sentiment", "sw") \
+  .setInputCols(["document", "sentence_embeddings"]) \
+  .setOutputCol("class")
+
+sw_pipeline = Pipeline(stages=[document_assembler, tokenizer, normalizer, stopwords_cleaner, embeddings, embeddingsSentence, sentimentClassifier])
+
+light_pipeline = LightPipeline(sw_pipeline.fit(spark.createDataFrame([['']]).toDF("text")))
+
+result1 = light_pipeline.annotate("Nimechanganyikiwa sana siku hizi.")
```

Comment:
I haven't found in the Internet these sentences, what very often means that they are not 100% native language, that is a translation. Can you use real sentences from the dataset? https://github.com/Jinamizi/Swahili-sentiment-analysis

Reply:
@josejuanmartinez 
I changed the sentences